data/Dockerfiles/solr/Dockerfile

```diff
@@ -2,12 +2,13 @@ FROM solr:latest
 
 ADD solr_core /opt/solr/server/solr/dovecot
 
-RUN cp -R /opt/solr/example/files/conf/lang /opt/solr/server/solr/dovecot/ \
-    && cp /opt/solr/example/files/conf/{protwords.txt,stopwords.txt,synonyms.txt,currency.xml,elevate.xml} \
-       /opt/solr/server/solr/dovecot/
-
 USER root
 
+RUN cp -R /opt/solr/example/files/conf/lang /opt/solr/server/solr/dovecot/conf/
+
+RUN ["/bin/bash", "-c", "cp /opt/solr/example/files/conf/{protwords.txt,stopwords.txt,synonyms.txt,currency.xml,elevate.xml} /opt/solr/server/solr/dovecot/conf/"]
+RUN mkdir -p /opt/solr/server/solr/dovecot/data/index
```

Comment:
This can't work. You're mounting a volume to /opt/solr/server/solr/dovecot/data. If the _index_ directory needs to be created manually (and is not automatically created when SOLR starts up), you need to create an entrypoint script that does it.